tests/subsys/suit/ipuc/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(integration_test_ipuc)
+include(../cmake/test_template.cmake)
+
+target_compile_options(suit_memory_layout_interface INTERFACE -DSDFW_UPDATE_AREA_ADDR=0x1000)
+target_compile_options(suit_memory_layout_interface INTERFACE -DSDFW_UPDATE_AREA_SIZE=0xFFFFE000)
```

Comment:
Isn't it a little too much?

Reply:
That is sufficient for posix tests, and, at the same time, do not generate false positive errors. The issue is that, in case of posix, an address coming from component ID is unpredictable. So it is hard to narrow this expected range.